lib/dartino/launch_dartino.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:async';
+
+import 'package:atom/node/fs.dart';
+import 'package:atom/node/process.dart';
+import 'package:logging/logging.dart';
+
+import '../atom.dart';
+import '../launch/launch.dart';
+import '../projects.dart';
+import '../state.dart';
+
+final Logger _logger = new Logger('atom.dartino_launch');
+
+class DartinoLaunchType extends LaunchType {
+  static void register(LaunchManager manager) =>
+      manager.registerLaunchType(new DartinoLaunchType());
+
+  DartinoLaunch _lastLaunch;
+
+  DartinoLaunchType() : super('dartino');
+
+  bool canLaunch(String path) {
+    DartProject project = projectManager.getProjectFor(path);
+    if (project == null || !project.isDartinoProject()) return false;
+    if (!analysisServer.isExecutable(path)) return false;
+    return true;
+  }
+
+  List<String> getLaunchablesFor(DartProject project) {
```

Comment:
Yes it will ... see note 10 lines below. This is temporary until I can add support to analysis server. Returning all the files is not an issue at this point because Dartino projects are small. Will update this method once analysis server has the appropriate support for Dartino.